apprentice_teacher.rb

```diff
@@ -1,5 +1,12 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
+require_relative 'institutional_learning'
+require_relative 'salary'
+# require_relative 'teach'
+
+class ApprenticeTeacher < Salary 
```

Comment:
The way to read this class definition is: `"An apprentice teacher IS A Salary"`.  Use "IS A" to check an inheritance relationship.  

Ultimately we are looking for:

- An Apprentice Teacher IS A Teacher
- A Senior Teacher IS A Teacher
- A Teacher IS A Person
- A Student IS A Person

How can you make that happen?